docs/guide/javaee/jmx.adoc

```diff
@@ -1,6 +1,6 @@
 ### JMX
 
-Management support is provided through the http://camel.apache.org/jmx.html[jmx,window=_blank] component which integrates with the {wildfly} JMX subsystem.
+You can provide the management support through the http://camel.apache.org/jmx.html[JMX,window=_blank] component which integrates with the {wildfly} JMX subsystem.
```

Comment:
Change 'You can provide the management support' to 'You can provide management support'.

Reply:
Fixed, as suggested.